man/os-release.xml

```diff
@@ -218,12 +218,16 @@
 
       <varlistentry>
         <term><varname>HOME_URL=</varname></term>
+        <term><varname>DOCUMENTATION_URL=</varname></term>
         <term><varname>SUPPORT_URL=</varname></term>
         <term><varname>BUG_REPORT_URL=</varname></term>
         <term><varname>PRIVACY_POLICY_URL=</varname></term>
 
         <listitem><para>Links to resources on the Internet related to 
-        the operating system. <varname>HOME_URL=</varname> should refer
+        the operating system.
+        <varname>DOCUMENTATION_URL=</varname> should refer to the main
+        documentation page for this operating system.
+        <varname>HOME_URL=</varname> should refer
```

Comment:
hmm, the order is a bit weird... I think the list above (the one with the `<term>`) and the explanation here below (the one within the `<listitem>`) really should have the same order, and probably in the order of "importance"...

Hence I think listing HOME_URL first and DOCUMENTATION_URL second would make most sense, i.e. follow the order you specified in the `<term>` list also in the explaining paragraph...

Reply:
Agreed; I thought I did that, but I was off-by-one...

New push fixes this.